src/rules/converters/no-unused-expression.ts

```diff
@@ -0,0 +1,15 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoUnusedExpression: RuleConverter = tslintRule => {
+    return {
+        rules: [
+            {
+                ...(tslintRule.ruleArguments.length !== 0 &&
+                    tslintRule.ruleArguments.includes("allow-tagged-template") && {
+                        ruleArguments: [{ allowTaggedTemplates: true }],

```

Comment:
TSLint also has `allow-fast-null-checks` and `allow-new` options that'll need to be accounted for.

Reply:
Hi, i didnt found the equivalent option for allow-fast-null-checks and allow-new in ESLint's rule (no-unused-expressions). Do you suggest some modification? Include notices?

Reply:
Yup! If there are options without equivalents, `notices` is the way to go.

Reply:
Hi, I included notice for options that ESLint not support.